src/modules/token/token.repository.ts

```diff
@@ -0,0 +1,39 @@
+import { Injectable } from '@nestjs/common';
+import { DataSource, Repository } from 'typeorm';
+import { TokenEntity } from '~/entities/token.entity';
+
+@Injectable()
+export class TokenRepository extends Repository<TokenEntity> {
```

Comment:
its somenthing over complicated, I'm not sure I understand why token should be stared in db for this project?

Reply:
I removed the Token module and entity.